cmd/version/git_native_disabled.go

```diff
@@ -0,0 +1,7 @@
+//go:build !gitenabled
+
+package version
+
+func isGitEnabled() bool {
```

Comment:
This should be in the `cautils` package.
This way we have the checks all in one place.

Reply:
done, pushed